src/main/java/org/mifos/processor/bulk/zeebe/worker/AuthorizationWorker.java

```diff
@@ -74,6 +76,9 @@ public void setup() {
 
     private HttpStatus invokeBatchAuthorizationApi(String batchId, AuthorizationRequest requestPayload, String clientCorrelationId) throws JsonProcessingException {
         logger.info("Calling auth API");
+        if (requestPayload.getAmount() == null || requestPayload.getAmount().isEmpty()) {
```

Comment:
can we use a  util function to check the empty condition( eg: SrtingUtils.isNotBlank)

Reply:
done